docs/Routing101.md

```diff
@@ -0,0 +1,30 @@
+## Routing for React apps
+
+### Production
+The production deployment of the vet-website front end consists of static HTML, CSS, and JS assets. We have an nginx server that serves those static assets and does a little bit of extra route handling for our single page React apps.
+
```

Comment:
Is it worth noting at all anything about S3?

Reply:
Wouldn't hurt